### ISSUE TITLE
Some users struggle to find information about support/opposition for a candidate

### ISSUE THREAD

**Issue by nickykrause** [18F/fec-testing#66]
In recent testing, users have been struggling with this question:

> As you may know, different groups of people can spend money during an election to try to help or to prevent a specific candidate from winning. Can you find the names of some groups that were trying to help or hurt Hillary Clinton during the 2016 presidential election?

The idea for this question comes from our past research which suggests that people are often looking for data on candidates that are of interest to them, especially in big races like the presidential elections, and that people seem to be curious/concerned about PAC spending.

Our assumption was that people would often initiate their searches by looking up the candidate, so we put information about relevant support/opposition IEs as its own tab on the candidate profile, e.g.:

<img width="1220" alt="screen shot 2017-05-22 at 10 07 29 am" src="https://cloud.githubusercontent.com/assets/1450173/26315341/800c3b26-3ed6-11e7-8fa7-e04716d17a00.png">

**However, here are the recent results:** 

> Users have been struggling with this task. During this test, two of the users failed and one succeeded indirectly by muddling through to the IE table. None of them thought to look up Hillary Clinton's candidate profile in order to find out who supported and opposed her. Similarly, in the previous test, two of the users looked up Clinton's committees instead of Clinton, and therefore couldn't find the support/oppose info. One user explained that it doesn't seem intuitive to look up a person to see who opposes them because she expects that it'll only contain info about people who support them.

Basically, we have tested this question in 2 of the 3 most recent rounds of testing, with a total of 7 users. **Of the 7 users who were asked this question, 4 of them struggled to answer it and did not seem inclined to go to a Candidate page for this kind of information.** Notably, these 4 users were not novices (they were moderate-level-to-advanced users)

There seems to be some signal here that, at least among non-novice users, information about spending to support/oppose a specific candidate is tricky to find.

**Comment by nickykrause** [comment]
Additional data point from testing today: One advanced user (a PhD candidate and researcher who has been working with FEC data for several years) had no trouble whatsoever when she was asked:

> There is a senator named Tammy Baldwin. Imagine that you want to know the names of some groups who are trying to prevent her from being re-elected. Can you find that information?

This was a new phrasing of the question designed to test the findability of this information. The user went an expected path and was successful: CFD > search tammy baldwin (no typeahead) > select senate candidate from results > candidate profile > spending by others to support or oppose.

**Comment by nickykrause** [comment]
^ This means that 4 of 8 users struggled to find the information, and it muddles the idea that the difficulty is typical of advanced users. The different phrasing of the question may also have made a difference.
